framework-core/src/main/java/org/daisy/pipeline/script/impl/parser/StaxXProcScriptParser.java

```diff
@@ -464,13 +464,13 @@ else if (elemsToIgnore == 0) {
 									else if (role.equals(Values.NAME)) {
 										reader.next();
 										dHolder.mShort = reader.peek()
-												.asCharacters().getData();
+												.asCharacters().getData().replaceAll("\\s\\s+","");
 									}
```

Comment:
@rdeltour  we wanted to avoid having repeated \s in sequence, as it disrupts the alignment in the GUI. (i.e. manual indentation after a \n ) See https://github.com/daisy/pipeline-issues/issues/467
I think the regexp is ok, it could be improved, but before considering that we could think of using some kind of text processing language for the documentation.